### ISSUE TITLE
Variable <unknown_object>.close(100825, -2147483648) not set before reading it

### ISSUE THREAD

**Issue by meseta** [zbanack/shampoo#5]
On trying to run `shampoo_canvas_close("canvas_name", true)` we get the error:

```
Variable <unknown_object>.close(100825, -2147483648) not set before reading it.
 at gml_Script_shampoo_canvas_close (line 90) - 	if (instant == true) canvas.close();
```

It looks like this line should read `canvas.free()` rather than `canvas.close()`,

Would you be open to accepting contributions from myself and @ArmyArmbuster (from over in #3) as we use shampoo in our project and encounter issues?

Also is this place still the best place to submit bug reports?

**Comment by zbanack** [comment]
Hi, thanks for the report. And yes! Please feel free to submit any issues or contributions here.

I'll get as many included in the next update—which I've delayed briefly due to my schedule.

**Comment by meseta** [comment]
cool! since there's no code in this repo, I'll send in git patches as issues

**Comment by zbanack** [comment]
Fixed in 0.9.4, available now on itch and the YYG Marketplace!
